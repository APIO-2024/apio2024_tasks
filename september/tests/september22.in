txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 0 0 0 2 2 4 4 6
7 5 3 8 9 4 6 2 1
10 1
0 1 0 1 0 1 4 3 5
9 2 8 5 4 7 3 1 6
10 1
0 1 2 3 1 5 3 1 5
9 4 6 8 5 7 2 3 1
10 1
0 0 1 3 4 3 2 4 4
8 6 7 5 9 2 4 3 1
10 1
0 1 0 1 3 2 3 6 7
1 3 8 2 9 7 5 4 6
10 1
0 1 2 3 0 3 6 0 5
4 8 7 5 9 6 3 1 2
10 1
0 1 1 2 3 2 1 5 0
3 2 1 7 5 4 9 6 8
10 1
0 1 2 1 0 0 4 6 7
8 9 5 6 7 3 4 2 1
10 1
0 1 1 2 3 1 5 0 6
9 7 8 6 5 2 1 4 3
10 1
0 0 2 1 0 0 5 7 8
4 6 9 1 2 3 8 7 5
10 1
0 0 1 1 2 4 0 0 6
2 7 5 3 8 6 9 4 1
10 1
0 1 0 3 1 0 2 7 5
9 8 6 4 5 7 2 1 3
10 1
0 1 2 3 0 3 1 6 4
8 7 5 6 9 4 3 2 1
10 1
0 1 2 1 2 3 5 7 4
8 4 9 2 6 5 7 3 1
10 1
0 0 2 1 1 0 1 0 1
3 6 9 2 8 4 5 7 1
10 1
0 0 0 3 1 5 5 3 7
4 8 2 6 9 3 5 1 7
10 1
0 0 2 0 1 5 5 3 4
9 7 6 8 4 5 3 1 2
10 1
0 1 1 0 2 1 6 1 6
4 5 1 2 6 7 3 9 8
10 1
0 1 2 1 3 3 6 3 6
7 8 5 4 9 3 6 2 1
10 1
0 1 2 3 4 5 4 3 0
9 6 8 5 7 4 3 2 1
40 1
0 1 0 1 0 1 2 1 8 6 4 6 0 8 4 9 4 11 8 19 16 17 15 17 23 10 24 23 10 2 24 21 26 29 10 34 29 5 11
13 12 39 7 38 3 36 31 20 28 14 25 24 27 35 18 30 5 21 32 2 11 37 33 23 19 10 34 29 26 15 16 22 6 9 8 4 1 17
40 1
0 1 1 2 0 3 0 1 0 0 2 11 6 13 0 13 1 12 8 14 2 15 13 16 16 3 12 12 19 7 30 12 12 1 26 10 18 31 16
22 4 35 33 34 36 38 17 9 26 39 10 21 25 20 5 23 32 31 28 37 24 16 15 30 27 14 18 12 7 11 2 29 13 6 3 19 8 1
40 1
0 0 0 0 4 1 4 3 0 5 4 7 5 13 6 13 10 14 3 11 0 14 2 5 6 17 23 22 19 5 13 28 15 6 14 8 26 26 5
21 12 31 37 20 18 38 25 29 27 23 33 2 34 24 9 16 11 36 8 15 30 7 19 32 28 26 6 22 39 3 17 35 1 10 14 5 13 4
40 1
0 0 0 3 0 1 5 3 6 5 8 8 5 5 10 15 1 1 15 11 1 2 1 0 4 20 7 4 17 4 19 6 0 29 26 4 19 31 21
16 25 22 18 37 35 28 24 39 30 36 34 27 29 21 13 26 7 20 14 33 2 11 4 32 17 8 12 6 9 38 3 31 19 23 1 5 10 15
40 1
0 1 2 3 1 2 4 4 2 2 6 5 0 0 0 12 7 0 4 17 8 11 18 20 1 13 21 24 15 13 30 1 31 29 25 3 1 34 21
35 19 9 36 23 25 14 16 18 22 26 10 27 37 11 32 39 28 21 12 33 8 31 5 24 38 34 29 30 15 20 13 17 6 4 7 3 2 1
100 1
0 0 1 2 3 4 5 0 7 0 3 10 7 1 12 15 12 15 10 15 8 6 21 4 20 21 1 9 15 13 30 15 0 14 30 9 1 32 18 12 12 19 41 42 17 14 12 37 40 42 37 9 24 30 7 7 20 21 33 36 4 54 39 14 54 34 34 38 42 4 63 19 46 53 46 26 55 39 65 77 48 35 2 79 69 76 26 13 14 3 48 65 57 74 38 20 37 56 51
23 81 72 31 83 84 16 96 75 25 89 28 47 52 29 98 64 71 85 79 94 59 50 69 92 68 93 57 90 60 20 88 63 67 97 74 82 66 65 11 62 45 80 91 58 73 22 61 77 43 55 46 95 34 53 54 41 48 17 70 24 14 36 35 78 38 99 30 18 9 33 39 13 44 51 32 56 27 87 37 15 4 2 19 42 6 86 7 49 76 5 26 21 12 8 10 40 1 3
1000 1
0 0 2 1 1 0 2 7 2 0 8 11 0 0 0 6 16 9 6 14 6 0 11 17 4 17 17 5 8 3 20 29 18 5 11 9 7 17 9 1 32 25 1 29 40 36 29 46 5 22 39 8 49 11 27 25 14 9 48 12 3 58 52 25 3 10 31 2 65 9 7 36 1 26 70 32 45 13 70 26 41 7 72 65 27 58 30 58 71 76 71 75 47 80 16 56 68 11 4 75 40 86 52 43 73 89 77 68 34 18 89 26 17 93 48 36 10 52 95 76 25 75 24 72 46 62 85 101 107 105 61 19 53 101 43 67 20 39 110 77 38 0 86 112 141 1 12 147 83 28 42 57 130 139 96 85 44 16 58 75 50 131 36 108 142 83 100 113 126 71 118 54 106 30 15 169 42 94 51 160 155 148 79 30 155 141 90 25 105 172 63 121 39 98 67 99 66 197 126 114 125 29 52 95 200 111 125 174 46 94 206 36 64 50 35 157 163 179 25 109 48 77 119 23 73 111 215 123 151 95 99 13 22 13 106 42 148 88 196 230 102 130 51 152 196 166 160 136 100 175 34 6 184 182 240 196 111 25 143 83 125 81 147 19 87 209 224 0 234 24 54 198 4 118 216 81 12 55 31 153 102 113 178 152 210 240 281 240 256 170 110 60 21 35 228 14 248 252 196 187 8 95 220 35 183 133 36 286 199 149 58 98 307 18 251 164 51 241 279 144 318 143 229 86 218 36 122 44 325 46 282 324 198 159 307 10 259 169 147 32 221 22 205 101 140 273 249 241 172 225 100 9 83 16 122 176 215 153 326 116 28 184 346 330 268 268 57 328 100 355 189 226 32 136 175 187 267 300 142 203 234 348 38 49 257 326 65 1 35 359 106 336 53 145 66 73 82 343 176 36 249 194 105 33 188 279 184 136 50 378 185 100 26 211 271 181 51 33 388 22 61 357 117 194 198 233 177 62 18 48 229 382 9 276 76 303 345 325 350 321 192 167 253 303 161 335 227 391 269 236 402 251 235 12 126 112 287 390 328 205 387 128 70 433 46 388 310 196 408 134 301 333 370 325 350 161 69 218 90 201 172 131 198 301 205 448 57 415 482 188 277 290 313 302 155 282 26 67 7 229 258 226 334 311 10 280 378 190 303 282 183 113 5 342 358 480 76 329 336 319 5 174 490 21 168 444 45 210 470 244 63 386 388 35 115 260 491 48 113 334 67 535 240 390 381 72 373 268 100 95 120 502 98 468 483 235 99 237 458 496 129 43 421 438 277 129 458 381 295 473 549 124 156 216 494 402 142 360 255 42 358 555 47 260 30 236 302 5 318 487 492 289 317 458 528 26 517 229 125 280 381 589 337 271 596 270 147 338 503 595 462 161 353 181 373 97 308 282 14 449 2 365 19 404 230 481 503 361 260 12 574 363 370 200 415 589 174 486 561 283 125 606 211 35 516 196 36 118 49 212 221 298 365 386 41 228 629 644 328 458 407 605 347 287 140 418 210 138 339 201 492 137 554 135 445 85 423 525 444 635 39 75 225 57 627 236 310 621 518 217 570 278 554 624 339 115 501 27 650 655 657 698 546 576 64 127 597 581 707 274 396 406 228 274 272 695 543 562 256 490 361 554 103 251 635 473 126 637 328 715 376 709 566 429 500 708 44 108 408 695 406 46 184 377 308 520 574 196 60 303 545 133 368 602 710 631 177 354 172 393 20 435 84 86 430 675 441 583 666 576 720 710 126 771 378 180 0 713 503 133 462 260 764 593 729 101 25 375 99 58 381 89 398 750 707 212 357 501 112 646 98 158 31 429 404 431 243 626 715 422 525 271 552 19 806 522 401 190 434 510 602 425 543 674 365 766 511 662 471 525 55 515 559 338 354 48 568 297 571 39 220 758 146 619 775 569 368 526 341 580 476 670 656 794 387 232 714 269 254 170 297 647 707 160 793 465 628 453 669 603 475 760 620 555 289 507 845 246 327 85 687 751 649 18 730 429 671 278 592 99 617 191 727 360 537 508 688 263 244 862 851 605 281 571 613 289 426 267 516 443 406 220 84 265 217 25 446 880 687 108 600 773 743 417 27 542 681 174 804 414 430 3 796 622 599 360 202 504 810 840 109 252 284 341 722 459 447 494 914 350 873 437 562 629 301 310 651 665 414 781 822 375 47 201 848 400 366 18 478 467 62 614 209 412 263 366 6 412 792 408 264 955 77 11 931 480 915 126 441 518 858 986 531 664 566 231 757 119 128
813 798 630 755 460 452 812 990 935 413 207 959 699 820 843 582 586 488 466 828 888 807 865 409 744 513 653 697 788 652 770 811 954 419 896 909 916 294 957 984 940 680 717 891 735 859 856 367 978 223 320 598 474 442 590 994 484 420 567 965 369 952 949 763 847 440 838 165 983 700 579 679 968 840 944 768 702 933 963 304 875 886 808 996 980 802 898 784 541 563 395 712 824 816 658 711 985 799 877 995 372 966 969 905 787 222 991 532 299 690 999 740 868 534 785 907 754 351 364 869 644 314 917 893 951 677 874 863 845 662 379 910 78 321 239 487 374 801 651 682 956 349 639 132 654 306 833 864 960 718 247 661 913 664 897 693 834 852 981 967 250 685 850 819 423 330 839 397 930 924 498 288 857 352 633 659 861 737 640 961 691 749 421 396 777 732 461 450 783 669 556 550 881 650 846 977 873 478 426 879 889 853 733 573 939 527 454 911 922 443 392 457 927 947 495 561 815 920 810 557 878 723 987 424 204 519 728 793 848 989 870 899 522 997 453 377 736 173 292 908 673 287 803 587 827 156 773 500 275 479 656 972 929 979 814 405 266 60 731 74 510 555 594 580 117 775 604 499 643 689 613 524 389 514 523 577 692 394 592 705 841 686 678 831 584 614 844 925 821 214 753 508 103 670 208 817 655 154 619 340 796 782 575 759 792 745 139 493 832 789 309 892 805 632 158 558 626 854 376 146 506 636 688 764 305 570 142 193 439 371 769 342 706 642 970 855 434 988 734 830 648 918 243 608 950 890 383 512 539 872 566 860 974 937 264 761 313 59 762 411 829 721 601 145 726 818 414 746 696 932 760 467 902 167 291 422 261 427 958 447 674 248 297 800 703 464 936 921 993 428 825 941 904 92 246 435 456 895 356 569 708 285 882 600 953 469 719 432 171 433 255 336 552 489 964 923 938 637 928 748 296 625 525 804 716 401 202 849 518 293 547 774 926 231 565 213 195 976 776 316 791 481 505 605 780 743 634 665 583 497 790 971 429 681 412 942 571 477 150 884 511 129 620 581 779 219 578 254 698 233 110 387 104 553 684 538 842 982 709 236 529 837 212 526 645 338 21 211 602 144 482 473 606 174 189 329 725 403 107 585 668 69 190 560 496 485 206 546 407 515 564 611 786 451 112 646 183 353 295 416 623 894 341 752 362 621 501 962 795 382 344 912 232 327 483 739 903 370 975 756 638 591 596 549 914 368 676 258 548 408 631 707 400 599 858 245 867 355 186 915 37 180 934 946 741 704 992 622 365 331 471 729 530 836 823 701 322 551 822 901 751 267 393 666 224 127 386 906 438 887 283 253 973 289 366 772 136 531 263 227 945 559 826 727 449 794 256 544 470 217 612 141 444 265 135 133 866 747 446 695 178 672 765 67 883 660 242 876 778 900 276 683 521 722 335 694 182 137 568 509 649 436 191 168 455 384 610 459 851 417 880 738 542 535 835 298 593 138 948 63 312 998 536 363 809 317 607 554 572 238 494 520 955 758 237 516 862 164 657 465 589 486 354 348 199 724 595 209 23 134 647 301 507 398 574 122 663 714 148 87 307 82 616 406 244 885 562 347 410 268 545 603 448 931 281 576 627 318 270 221 463 269 278 797 97 504 671 399 319 172 378 629 609 323 418 375 332 781 163 162 79 528 667 160 502 437 124 710 713 503 120 617 492 228 308 235 85 339 185 181 175 290 226 597 170 325 943 343 641 249 123 311 360 635 91 300 274 871 402 166 588 116 24 84 730 806 537 43 176 279 624 540 462 476 491 19 533 65 767 346 425 194 757 98 742 64 441 273 385 131 715 169 177 334 119 187 216 750 315 472 380 357 192 468 241 218 475 345 61 128 140 100 115 55 431 102 302 277 391 53 618 628 159 771 90 324 27 31 75 251 118 229 210 615 184 205 303 252 986 480 919 157 458 720 88 282 215 15 196 106 373 49 766 328 44 543 381 361 430 20 28 155 203 687 201 99 260 262 101 151 38 404 280 220 126 48 234 284 350 415 109 333 125 46 34 52 272 30 108 200 81 70 68 358 3 310 675 152 257 198 41 337 42 225 390 490 359 153 130 121 326 62 33 149 114 113 105 93 388 259 179 35 94 188 271 111 96 56 286 89 445 143 51 197 240 39 47 83 71 230 80 95 26 72 17 25 161 86 16 57 58 66 50 13 77 22 10 18 36 147 12 45 6 14 4 517 54 9 5 11 73 40 1 76 8 32 29 7 2
100 1
0 0 0 0 2 0 4 6 1 3 0 11 9 9 0 11 10 6 15 6 17 18 18 20 11 8 26 21 10 16 23 6 17 27 12 19 1 6 26 8 33 16 29 24 20 16 2 25 1 17 6 33 2 13 31 28 35 19 54 54 47 33 62 5 47 58 3 1 14 27 19 41 23 55 51 50 51 6 61 56 41 38 64 81 13 44 50 84 63 20 71 51 58 87 58 0 58 24 52
43 74 42 49 77 67 78 94 37 91 40 76 53 32 98 92 69 90 7 68 22 72 87 75 59 83 95 80 99 97 86 52 88 57 36 30 34 84 44 48 71 50 93 89 45 24 56 14 85 55 82 46 60 39 35 16 79 31 81 64 73 61 41 20 29 54 4 5 23 13 25 96 51 38 63 9 18 28 62 33 12 11 21 65 47 1 66 2 58 17 10 19 3 15 70 26 27 8 6
40 1
0 1 1 3 3 5 1 3 5 1 5 10 6 11 11 13 6 1 2 14 14 8 11 11 6 0 6 24 20 5 4 1 22 33 26 7 4 28 2
9 38 18 39 15 23 29 36 37 25 12 31 30 32 7 17 19 35 20 16 34 2 27 26 33 28 4 21 22 10 14 8 13 6 24 11 5 3 1
40 1
0 1 2 0 2 1 5 3 1 2 6 1 7 1 1 1 11 7 7 19 3 9 2 3 8 7 21 26 12 3 8 20 32 23 1 16 13 28 12
14 10 24 31 34 18 27 21 23 38 30 33 29 17 15 22 35 39 32 3 9 11 13 4 6 8 25 36 37 20 19 7 28 2 16 5 1 12 26
40 1
0 1 1 3 1 1 4 2 2 5 0 7 2 4 4 5 11 0 13 9 20 12 17 4 7 17 26 11 20 4 14 8 9 18 28 31 11 35 30
32 33 37 29 23 6 19 22 24 25 36 16 34 13 18 39 10 12 27 8 31 5 38 21 26 20 14 7 9 2 15 17 28 35 30 4 11 3 1
40 1
0 1 1 1 3 5 2 6 1 7 2 2 5 6 2 14 16 11 5 10 0 1 21 15 9 14 13 1 1 17 16 20 8 12 13 17 33 8 1
32 37 39 23 28 24 38 27 4 19 29 15 20 26 35 18 36 21 11 30 31 22 13 25 9 34 10 17 33 8 7 16 12 2 14 5 6 3 1
40 1
0 1 2 3 1 0 1 6 7 8 1 8 7 12 0 13 16 14 3 12 12 10 2 0 24 24 3 8 3 2 4 6 1 6 24 4 33 9 7
29 21 26 30 32 15 27 35 17 37 38 23 39 28 25 18 24 16 34 20 9 5 36 11 13 7 22 33 14 12 19 10 8 6 31 4 3 2 1
10 1
0 0 0 3 1 2 1 4 0
9 8 5 7 2 4 1 6 3
10 1
0 1 0 2 4 5 0 0 3
4 7 8 3 9 6 2 1 5
10 1
0 0 2 3 0 5 1 7 4
6 7 8 9 1 4 5 2 3
10 1
0 1 0 3 3 5 3 2 7
6 5 4 9 2 7 8 3 1
10 1
0 1 1 2 1 3 4 2 6
7 5 8 4 9 6 2 3 1
10 1
0 1 2 0 0 1 1 5 7
9 4 6 3 7 8 1 5 2
10 1
0 0 2 3 4 2 5 2 3
1 8 9 5 7 6 2 3 4
10 1
0 1 0 1 3 2 1 1 4
2 6 5 9 7 1 8 4 3
10 1
0 0 2 2 0 0 6 5 6
2 6 8 5 1 7 9 3 4
10 1
0 0 1 1 0 3 3 7 5
4 7 6 2 8 9 1 5 3
10 1
0 0 0 2 0 5 0 0 1
8 4 3 9 1 2 7 6 5
10 1
0 0 2 2 0 2 5 2 1
4 7 1 9 6 2 3 8 5
10 1
0 1 0 3 1 4 5 1 1
6 2 3 8 1 7 9 4 5
10 1
0 0 0 1 0 2 1 7 1
9 5 6 2 4 3 8 7 1
10 1
0 0 1 0 4 5 1 0 4
6 8 4 9 3 1 5 7 2
10 1
0 0 2 2 0 4 5 4 5
1 7 6 9 3 5 8 4 2
10 1
0 0 1 1 1 2 4 4 5
9 7 5 6 3 2 1 4 8
10 1
0 1 1 1 1 1 5 4 1
8 3 7 4 2 9 5 1 6
10 1
0 0 0 3 0 0 2 1 6
7 8 2 9 6 5 1 4 3
10 1
0 0 2 0 2 0 6 0 0
7 4 8 1 2 3 9 6 5
