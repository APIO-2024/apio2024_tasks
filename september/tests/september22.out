7ckgnn4wyi495puj3ibqf81dqvapyv6b
9
7
8
9
1
7
1
9
6
8
6
9
9
4
9
7
9
3
8
9
32
39
38
35
38
88
996
98
39
23
38
38
39
6
1
7
7
9
7
6
6
1
4
9
6
3
9
4
9
7
8
9
5
